its/pom.xml

```diff
@@ -50,22 +50,17 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.13.1</version>
+      <version>4.13.2</version>
```

Comment:
It's painful to update these 3x. What about extracting it as a property to the root pom, just like the other versions we maintain here https://github.com/SonarSource/sonar-dotnet/blob/8ab4bcc44e888f5d823182bfa387f81b8125052a/pom.xml#L52-L62 ?

Therese are test-only dependencies. And there should be no harm in mentioning them in the root